app/src/main/java/pl/temomuko/autostoprace/ui/staticdata/PartnerDrawables.kt

```diff
@@ -0,0 +1,42 @@
+package pl.temomuko.autostoprace.ui.staticdata
+
+import pl.temomuko.autostoprace.R
+
+object PartnerDrawables {
+
+    @JvmStatic val strategic = listOf(
```

Comment:
Should be STRATEGIC
https://kotlinlang.org/docs/reference/coding-conventions.html#property-names